packages/packages/commonwell-cert-runner/src/payloads.ts

```diff
@@ -108,3 +110,59 @@ export const mergePatient = {
   ],
   details: secondaryDetails,
 };
+
+// ORGANIZATION
+const appendOrgId = nanoid.customAlphabet("1234567890", 18)();
+const shortName: string = uniqueNamesGenerator({
+  dictionaries: [adjectives, colors, animals],
+  separator: "-",
+  length: 3,
+});
+
+export const organization = {
+  organizationId: `urn:oid:${commonwellOID}.${appendOrgId}`,
+  homeCommunityId: `urn:oid:${commonwellOID}.${appendOrgId}`,
+  name: shortName,
+  displayName: shortName,
+  memberName: "Metriport",
+  type: "Hospital",
+  patientIdAssignAuthority: `urn:oid:${commonwellOID}.${appendOrgId}`,
+  securityTokenKeyType: "BearerKey",
+  isActive: true,
+  locations: [
+    {
+      address1: "1 Main Street",
+      address2: "PO Box 123",
+      city: "Denver",
+      state: "CO",
+      postalCode: "80001",
+      country: "USA",
+      phone: "303-555-1212",
+      fax: "303-555-1212",
+      email: "here@dummymail.com",
+    },
+  ],
+  technicalContacts: [
+    {
+      name: "Technician",
+      title: "TechnicalContact",
+      email: "technicalContact@dummymail.com",
+      phone: "303-555-1212",
+    },
+  ],
+};
+
+export const thumbprint = "11E378F987D1C716B1FD5E08004E996AC806A9F1";
```

Comment:
is the SHA-1 fingerprint/thumbprint of the certificate - we can calculate this based on the configured cert. See: https://www.geeksforgeeks.org/node-js-x509-fingerprint-property/